src/main/drivers/flash_w25n01g.c

```diff
@@ -327,9 +330,15 @@ bool w25n01g_identify(flashDevice_t *fdevice, uint32_t jedecID)
 {
     switch (jedecID) {
     case JEDEC_ID_WINBOND_W25N01GV:
-        fdevice->geometry.sectors = 1024;      // Blocks
-        fdevice->geometry.pagesPerSector = 64; // Pages/Blocks
-        fdevice->geometry.pageSize = 2048;
+        fdevice->geometry.sectors = W25N01G_BLOCKS_PER_DIE;      // Blocks
```

Comment:
This should be in array eventually, but for this PR it's still ok

Reply:
Yes, as per the `flash_m25p16` driver if more devices are added.

Reply:
Done